rasa/nlu/utils/spacy_utils.py

```diff
@@ -9,6 +10,8 @@
 from rasa.shared.nlu.training_data.message import Message
 from rasa.nlu.model import InvalidModelError
 from rasa.nlu.constants import SPACY_DOCS, DENSE_FEATURIZABLE_ATTRIBUTES
+from rasa.shared.utils.io import raise_deprecation_warning
```

Comment:
```suggestion
rasa.shared.utils.io
```

According to our Python code conventions we don't import functions directly but reference them by their module.  https://www.notion.so/rasa/Python-Code-Conventions-26835ffd72484dd699f28fa693916d2e#5b63fd5269514270aaf61ab8303ae319